OasisDEX/oasis-borrow
# Identify tokens

# [Identify tokens](https://app.shortcut.com/oazo-apps/story/10588/identified-tokens-stored-in-database)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- adjusted `idenftifyTokens` observable
- added caching layer in db
  
## How to test 🧪
  <Please explain how to test your changes>

- use `identifyTokens$` in any place you want with tokens addresses, verify whether tokens not available in local config has been saved to db to avoid unnecesary api / rpc calls
